packages/pancake-profile-sdk/src/profile-sdk.ts

```diff
@@ -102,11 +101,12 @@ class PancakeProfileSdk {
         2: numberUsers,
         3: numberPoints,
         4: isJoinable,
-      } = await this.profileContract.methods.getTeamProfile(teamId).call();
+      } = await this.profileContract.getTeamProfile(teamId);
```

Comment:
Just double-checking that the typing of what's returned by the call is unchanged between web3 & ethers? 

Same Q goes for `getUserProfile`? 

Maybe that as no BNs are involved this is a cleaner migration than some of the others?